liwords-ui/src/lobby/seek_form.tsx

```diff
@@ -275,6 +292,15 @@ export const SeekForm = (props: Props) => {
     required: 'This field is required.',
   };
 
+  useEffect(() => {
+    if (usernameOptions.length === 0) {
+      const defaultOptions = tournamentID
+        ? presences.map((p) => p.username).filter((u) => u !== username)
+        : Object.values(friends).map((f) => f.username);
+      setUsernameOptions(defaultOptions);
+    }
+  }, [friends, presences, username, usernameOptions, tournamentID]);
```

Comment:
when the modal to create game is shown (whether Play a computer, Match a friend, or Create a game) the screen froze.
```
Warning: Maximum update depth exceeded. This can happen when a component calls setState inside useEffect, but useEffect either doesn't have a dependency array, or one of the dependencies changes on every render.
    in SeekForm (at gameLists.tsx:228)
```
this is because `defaultOptions` is always a new object, and `setUsernameOptions` re-triggers this effect.

(ps. feel free to merge once it works! thanks for this feature ❤️ )

Reply:
Thanks for all your hard work on this. The defaultOptions fix you mentioned above fixed this.

Reply:
Turns out this wasn't fixed. Infinite loop was caused by setting usernameOptions without a guard in an effect that had usernameOptions as a dependency. Fixed in master.